utils/nctl/docs/setup.md

```diff
@@ -0,0 +1,49 @@
+# nctl setup
+
+### Step 0 - pre-requisites.
+
+1. python3 + pip3.
+2. The casper-node software (https://github.com/CasperLabs/casper-node) cloned into YOUR_WORKING_DIRECTORY.
+
+### Step 1 - install pre-requisites.
+
+```
+# Supervisor - cross-platform process manager.
+python3 -m pip install supervisor
+
+# Rust toolchain - required by casper-node software.
+cd YOUR_WORKING_DIRECTORY/casper-node
+make setup-rs
+```
+
+### Step 2 - set local setup file.
+
+```
+cat >> $HOME/.casper-nctl <<- EOM
+
+# Activate nctl shell.
+. $(pwd)/casper-node/utils/nctl/activate
```

Comment:
Will that be correct if the user has done `cd YOUR_WORKING_DIRECTORY/casper-node` in the step above?

Reply:
1.  I just corrected this potential path issue. 
2. I am inclined **not** to include deprecating `run-dev.sh` & `stop-dev.sh` are in this PR too.